ts/components/screens/BaseScreenComponent.tsx

```diff
@@ -3,6 +3,7 @@ import { connectStyle } from "native-base-shoutem-theme";
 import mapPropsToStyleNames from "native-base/src/utils/mapPropsToStyleNames";
 import * as React from "react";
 
+import { ComponentProps } from "../../types/react";
```

Comment:
ComponentProps is already included in react-native. Can you try to use the one from react-native?

Reply:
Fixed in c2c58540